src/gsi/cplr_read_wrf_mass_guess.f90

```diff
@@ -1358,6 +1358,8 @@ subroutine read_wrf_mass_netcdf_guess_wrf(this,mype)
   !                            - add CV transform option on hydrometer variables
   !   2022-03-15  Hu  change all th2 to t2m and convert 2m temperature 
   !                        from potentionl to senseible temperature
+  !   2025-01-28  zhao   - added code to read wave height (howv) & wind gust (gust) 
+  !                        from intermediate binary file for the analysis in WRF-ARW based 3DRTMA
```

Comment:
Adding changelog entries to source code is no longer needed.  Version control along with well documented issues record who changed the code and what the change is for.  

It's fine to keep the new changelog entries added by this PR. I suggest that future PRs do not add changelogs to the source code.

Reply:
I removed this changelog entry as well as other new changelog entries introduced by this PR to GSI code. I will re-submit this PR soon.
Thank you for letting me know about this rule. I will keep it in mind.